app/views/users/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="users container">
+  <ul class="user_list col">
+    <% @users.each do |user| %>
+      <li class="user_item row">
+        <div class="u_img">
+          <img src=<%= user.photo %> alt="u_img">
+        </div>
+        <div class="user_cont">
+          <a href=<%= user_url(user.id) %>>
+            <div class="user_desc col">
+              <h2 class="user_name"><%= user.name %></h2>
+              <p class="user_post_n">Number of posts: <%= user.posts_count %></p>  
+            </div>
+          </a>
+        </div> 
+      </li>
+    <% end %>
+  </ul>
+</div>
```

Comment:
![image](https://github.com/BilalLiaquat7/blog_app/assets/93039359/1ff9b4fb-c90a-41e4-a85a-297f0c116168)

![image](https://github.com/BilalLiaquat7/blog_app/assets/93039359/8692e176-1e61-4203-a6f0-874adfed8fc1)

![image](https://github.com/BilalLiaquat7/blog_app/assets/93039359/a1d1d132-c9da-4ca0-9b73-bdc86aa868bb)

![image](https://github.com/BilalLiaquat7/blog_app/assets/93039359/c225f1d0-f4f4-4e2b-ad82-7cdffd629538)

- [ ] I am not able to see any of the view of your project :fearful: Kindly fix them because is I am not able to see any view work as expected :fearful: Kindly fix them to make your views work as expected :blush: